node/src/node/slave/fetcher.cpp

```diff
@@ -50,6 +50,9 @@ fetcher_t::watch() {
 
 void
 fetcher_t::on_read(const std::error_code& ec, size_t len) {
+    if (0 < len){
+        COCAINE_LOG_DEBUG(slave->log, "got slave output of %d bytes", len);
```

Comment:
That was a case, when the open named pipe gave output in 4k chunks. Still have to workaround it somehow.